def scope_test():
    x = 123

scope_test()
# print(x)

# The variable x is not defined in the global scope, so this will raise a NameError.